tests/fixtures/fill.html

```diff
@@ -6,6 +6,6 @@
     <title>Fill</title>
   </head>
   <body>
-    <input id="target"></input>
+    <input id="target" />
```

Comment:
Self closing tags don't exist in html

Reply:
But deno formatter complain about it.

Reply:
Could you change it to `<input id="target">`? I believe the formatter should stop complaining about it.

Reply:
```
> deno fmt --check
Error checking: astral\tests\fixtures\fill.html
  Syntax error (expected close tag) at file:///astral/tests/fixtures/fill.html:9:26
```

This could be unexpected behavior. I also found a [stackoverflow answer](https://stackoverflow.com/a/76231113) about this.

Reply:
Anyway currently deno 2 formatter reject both `<input id="target">` and `<input id="target"></input>`.

Reply:
Welp, okay I guess this is fine then.